src/Donors/Models/Donor.php

```diff
@@ -50,6 +52,7 @@ class Donor extends Model implements ModelCrud, ModelHasFactory
         'firstName' => 'string',
         'lastName' => 'string',
         'email' => 'string',
+        'phone' => 'string',
```

Comment:
Please update `TestDonor` & `TestDonorRepository` to reflect this change.

Reply:
Resolved in 2fd2065c9df9a2ef13924b2211ed650a3707b847 and f7a4c8528b62ba170796ad49d75b029c4601b182